_includes/footer.html

```diff
@@ -13,6 +13,9 @@ <h5>Social Media</h5>
     <a href="https://github.com/{{ site.contact.github }}/" target="_blank" rel="noopener noref">Github</a>
     <a href="https://medium.com/{{ site.contact.medium }}" target="_blank" rel="noopener noref">Medium</a>
   </div>
+  <div>
+    {% include donateButton.html %}
+  </div>
```

Comment:
The footer already includes navigation.html, so the Donate button appears twice:

![Donate button at the end of the Site Map column and in a column of its own](https://github.com/user-attachments/assets/3ef55c9f-6c62-4066-bc23-5ceb22e8e662)